platform/android/export/export.cpp

```diff
@@ -692,6 +692,8 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		int xr_mode_index = p_preset->get("xr_features/xr_mode");
 
+		String v2_plugins = p_preset->get("custom_template/v2_plugins");
```

Comment:
Are we actually keeping support for `v1` plugins after this PR?
If not, I think it might be better to drop the `v2` naming and just make it the de facto plugin system. Docs can be updated to describe the new plugin system only instead of both, and we can add a porting guide from Godot 3.[01] (`v0`? :)) and 3.2 plugins to 4.0+.

Reply:
If we agree about not keeping the `v1`/`v2` distinction in the codebase and just support the new system (at least for 4.0), then all references to `v2_plugins` should be changed to simply refer to `plugins`.